qutebrowser/mainwindow/treetabwidget.py

```diff
@@ -35,8 +35,6 @@ def __init__(self, win_id, parent=None):
         # root of the tab tree, common for all tabs in the window
         self.tree_root = Node(None)
         super().__init__(win_id, parent)
-        self.tabBar().tabMoved.disconnect(  # type: ignore
-            self.update_tab_titles)
```

Comment:
I remove this line because it cause crash on startup. Sadly I am not sure what this line does.

Reply:
It was intentionally added for performance in https://github.com/qutebrowser/qutebrowser/pull/4602/commits/b8d9622aa64324a4c96aed205a67e845d5fb2c11. The crash comes from you missing the other change at https://github.com/qutebrowser/qutebrowser/commit/b8d9622aa64324a4c96aed205a67e845d5fb2c11#diff-7c41aa2423e4249167ffe8cbb8680380L70-R70:
```diff
-       bar.tabMoved.connect(functools.partial(
-           QTimer.singleShot, 0, self.update_tab_titles))
+       bar.tabMoved.connect(self.update_tab_titles)
```

Reply:
Thanks for the pointer :smile: 

I introduce back the changes.